### PR TITLE
Add suffixes to typos from "b" to "ch"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4940,6 +4942,8 @@ behaviro->behavior
 behaviuor->behaviour
 behavoir->behavior
 behavoirs->behaviors
+behavoiur->behaviour
+behavoiurs->behaviours
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:4946]: Perhaps add
```
behavoiural->behavioural
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5826,6 +5839,7 @@ caclulates->calculates
 caclulating->calculating
 caclulation->calculation
 caclulations->calculations
+caclulator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5842]: Add plural?
```
caclulators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5909,24 +5923,42 @@ calbirator->calibrator
 calbirators->calibrators
 calcable->calculable
 calcalate->calculate
+calcalated->calculated
+calcalates->calculates
+calcalating->calculating
+calcalation->calculation
+calcalations->calculations
+calcalator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5931]: Plural again?
```
calcalators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -5909,24 +5923,42 @@ calbirator->calibrator
...
 calciulate->calculate
 calciulating->calculating
+calclate->calculate
+calclated->calculated
+calclates->calculates
+calclating->calculating
 calclation->calculation
+calclations->calculations
+calclator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5940]: Plural?
````
calclators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -5909,24 +5923,42 @@ calbirator->calibrator
...
 calcluate->calculate
 calcluated->calculated
 calcluates->calculates
+calcluating->calculating
+calcluation->calculation
+calcluations->calculations
+calcluator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5947]: Plural
```
calcluators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -5909,24 +5923,42 @@ calbirator->calibrator
...
 calclulate->calculate
 calclulated->calculated
 calclulates->calculates
 calclulating->calculating
 calclulation->calculation
 calclulations->calculations
+calclulator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5954]:
```
calclulators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -5909,24 +5923,42 @@ calbirator->calibrator
...
 calcualate->calculate
 calcualated->calculated
 calcualates->calculates
 calcualating->calculating
 calcualation->calculation
 calcualations->calculations
+calcualator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5961]:
```
calcualators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -5940,8 +5972,10 @@ calcualtor->calculator
 calcuate->calculate
 calcuated->calculated
 calcuates->calculates
+calcuating->calculating
 calcuation->calculation
 calcuations->calculations
+calcuator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5978]:
```
calcuators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -5955,24 +5989,50 @@ calculattions->calculations
 calculaution->calculation
 calculautions->calculations
 calculcate->calculate
+calculcated->calculated
+calculcates->calculates
+calculcating->calculating
 calculcation->calculation
+calculcations->calculations
+calculcator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5997]:
```
calculcators->calculators
```

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -5955,24 +5989,50 @@ calculattions->calculations
...
+calcultion->calculation
+calcultions->calculations
+calcultor->calculator
+calculuate->calculate
+calculuated->calculated
+calculuates->calculates
+calculuating->calculating
+calculuation->calculation
 calculuations->calculations
+calculuator->calculator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:6021]:
```
calculuators->calculators
```

**Review by DimitriPapadopoulos — COMMENTED** [review]
For all occurrences of:
```
...->calculator
```
I suggest adding the plural:
```
...->calculators
```
I started with a comment for each occurrence, but there are too many of them.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:4952]: Done. I added suffixes to the surrounding words.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**